src/content/blog/2024-09-30-dwmrc-and-ugrc-release-map-to-help-utahns-recycle-better.mdx

```diff
@@ -0,0 +1,30 @@
+
+These data products are part of a larger effort to improve the way we manage waste in the United States. According to the Environmental Protection Agency, the US generated about 4.9 pounds of municipal solid waste (MSW) per person per day in 2018 ([1](https://www.epa.gov/facts-and-figures-about-materials-waste-and-recycling/national-overview-facts-and-figures-materials)). That's a lot of cockroaches that will never truly disappear. As we work toward processing waste more efficiently (and producing less of it in the first place), projects like this one give us a clear understanding of where waste is coming from, where its going, and what systems could be improved. So, the next question is, how did we create these data products, and what do they mean for waste management efforts?
+
+Let's start with the process. First, UGRC extracts the data from DWMRC's SalesForce database, which houses the reporting forms sent in by municipal and state waste management facilities. Since every waste management facility in Utah is required to report their data on standardized forms, there was a large volume of data to work with, and more is added every week. 
+
+Once the tabular data are organized and cleaned, we use pandas scripts to perform calculations and gather statistics from the reports. These scripts run fully automated so that the statistics will be updated as new data are entered into the database. The fully automated process saves time, money, and helps to prevent human error.
+
+Even if you don't have a haunted vase from a long-lost relative lying around (perhaps we should all check?), there are plenty of reasons why having access to this data is a good thing. First and foremost, the map serves as a reference for where you can dispose of any other haunted articles you have in your possession and regular items such as paint cans, batteries, and furniture. The map also points out recycling centers and has been used by organizations such as the Utah Recycling Alliance to help spread awareness about recycling efforts ([2](https://www.utahrecyclingalliance.org/post/utah-s-new-recycling-data-initiative)). Just knowing where and how to recycle can help people to actually do it. 
+
+Another benefit of this project is transparency. Utahns deserve to know how the government is using taxpayer's dollars to promote effective waste management practices in our state. With these data products, the average person can see important metrics such as the percentage of waste materials that were recycled in Utah and how many gallons of used oil were collected for recycling. Having these metrics publicly available adds an extra layer of accountability for waste management facilities, DWMRC.
```

Comment:
```suggestion
Another benefit of this project is transparency. Utahns deserve to know how the government is using their tax dollars to promote effective waste management practices in our state. With these data products, the average person can see important metrics such as the percentage of waste materials that were recycled in Utah and how many gallons of used oil were collected for recycling. Having these metrics publicly available adds an extra layer of accountability for waste management facilities, DWMRC.
```

Also, the last sentence here feels like there was something more that was meant to go on there.